cobra/core/object.py

```diff
@@ -40,6 +40,19 @@ def id(self, value):
     def _set_id_with_model(self, value):
         self._id = value
 
+    @property
+    def annotation(self):
+        return getattr(self, "_annotation", None)
```

Comment:
Since the attribute `_annotation` is created on line 23, why do you expect it to not exist here?

```suggestion
        return self._annotation
```

Seems to be sufficient?

Reply:
Oh yaah, sorry about that. I'll change it. Thanks